lib/rails_semantic_logger/extensions/action_dispatch/debug_exceptions.rb

```diff
@@ -7,9 +7,11 @@ class DebugExceptions
 
     undef_method :log_error
     if (Rails::VERSION::MAJOR == 7 && Rails::VERSION::MINOR >= 1) || Rails::VERSION::MAJOR > 7
-      def log_error(_request, wrapper)
+      def log_error(request, wrapper)
         Rails.application.deprecators.silence do
-          level = wrapper.respond_to?(:rescue_response?) && wrapper.rescue_response? ? :debug : :fatal
+          return if !log_rescued_responses?(request) && wrapper.rescue_response?
```

Comment:
In https://github.com/reidmorrison/rails_semantic_logger/pull/179, there was a [concern](https://github.com/reidmorrison/rails_semantic_logger/pull/179#issuecomment-1484237876) about the `rescue_responses?` method not working in Rails 5 and 6, which led to this `response_to?` check being added. I don't believe it's necessary, however, because this is within a conditional that checks `(Rails::VERSION::MAJOR == 7 && Rails::VERSION::MINOR >= 1) || Rails::VERSION::MAJOR > 7`, so we can feel confident that we are in Rails 7.1+. Given that, I took the liberty of removing the `respond_to?` check